go.mod

```diff
@@ -39,6 +39,7 @@ require (
 	github.com/klauspost/compress v1.15.9
 	github.com/moby/locker v1.0.1
 	github.com/moby/sys/mountinfo v0.6.2
+	github.com/moby/sys/sequential v0.0.0-20220829095930-b22ba8a69b30
```

Comment:
No release yet, but I'll do a release once review of all PR's is looking good, and will update to v0.5.0 (once release) in a follow-up.